src/main/webapp/i18n/de/fileUploadExercise.json

```diff
@@ -7,6 +7,8 @@
                 "editLabel": "Datei-Upload-Aufgabe bearbeiten",
                 "importLabel": "Datei-Upload-Aufgabe importieren"
             },
+            "create": "Datei-Upload erstellen",
+            "import": "Datei-Upload importieren",
```

Comment:
Please remove this and use the existing translation instead (or remove the previous translation and make sure to replace it) 

Also it was `Datei-Upload-Aufgabe` before, why is the `Aufgabe` missing now?